run/markdown-preview/renderer/main.js

```diff
@@ -0,0 +1,50 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const express = require('express');
+const MarkdownIt = require('markdown-it');
+const xss = require('xss');
```

Comment:
Does markdown-it have built-in xss protection?

Reply:
@grayside Yes, per their [docs](https://github.com/markdown-it/markdown-it/blob/master/docs/security.md). I've removed xss from the code & the dependencies.